tests/unit/s2n_stuffer_network_order_test.c

```diff
@@ -19,7 +19,10 @@
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_mem.h"
 
-#include "stuffer/s2n_stuffer_network_order.c"
+#define SIZEOF_UINT24 3
```

Comment:
This is where the U24_SIZE could have been used. This is one of those things that doesn't really matter for this PR though. We can have this defined in a single location later if it is needed throughout more files.